feature-libs/product-configurator/rulebased/core/model/configurator.model.ts

```diff
@@ -100,6 +100,10 @@ export namespace Configurator {
     hideBasePriceAndSelectedOptions?: boolean;
   }
 
+  export interface ConfigurationWithOverview extends Configuration {
+    overview: Overview;
+  }
```

Comment:
@ChristophHi: i created this - its used in the components - do you think this is ok?
It extends the Configuration interface, but the overview property is mandatory, which is guaranteed by having a corresponding filter in the components observable.

Reply:
I don't like it too much. If you need the OV to be mandatory, why not have it directly as part of the method signatures where you are using ConfigurationWithOverview now?

Reply:
@Uli-Tiger But when I check it a little bit longer I understand why it helps. The suggestion that I made does not work because then we probably would also need to change the signatures of facade, actions and do changes to the effects. Let me think about it